go.mod

```diff
@@ -17,7 +17,6 @@ require (
 	github.com/google/go-github/v33 v33.0.0
 	github.com/google/gofuzz v1.2.0
 	github.com/gosuri/uitable v0.0.4
-	github.com/mattn/go-runewidth v0.0.13 // indirect
```

Comment:
To clarify can this be removed because it's never imported, as in second bulled point here https://golangbyexample.com/indirect-dependency-golang/?

fwiw some go mod graph deps
```
k8s.io/kubectl@v0.22.2 k8s.io/cli-runtime@v0.22.2
k8s.io/kubectl@v0.22.2 sigs.k8s.io/kustomize/kustomize/v4@v4.2.0
sigs.k8s.io/kustomize/kustomize/v4@v4.2.0 sigs.k8s.io/kustomize/cmd/config@v0.9.13
sigs.k8s.io/kustomize/cmd/config@v0.9.13 github.com/olekukonko/tablewriter@v0.0.4
github.com/olekukonko/tablewriter@v0.0.4 github.com/mattn/go-runewidth@v0.0.7
```